Dockerfile

```diff
@@ -12,37 +12,30 @@ RUN apk update
 RUN set -eux && \
     apk add --no-cache bash && \
     apk add --no-cache zip && \
-    apk add --no-cache apache-ant
+    apk add --no-cache apache-ant && \
+    apk add --no-cache fontconfig ttf-dejavu
+
 
 
 WORKDIR /usr/src
 
 RUN set -eux && \
     apk add --no-cache wget && \
-     wget -cO jython-installer.jar "http://search.maven.org/remotecontent?filepath=org/python/jython-installer/$JYTHON_VERSION/jython-installer-$JYTHON_VERSION.jar" && \
-     java -jar jython-installer.jar -s -t minimum -d "$JYTHON_HOME" && \
-     rm -fr "$JYTHON_HOME"/Docs "$JYTHON_HOME"/Demo "$JYTHON_HOME"/tests && \
-     rm -f jython-installer.jar && \
-     ln -sfv "$JYTHON_HOME/bin/"* /usr/local/bin/ && \
-     wget -cO jfreechart.zip "https://sourceforge.net/projects/jfreechart/files/1.%20JFreeChart/1.0.15/jfreechart-1.0.15.zip/download" && \
-     unzip jfreechart.zip && \
-     rm -f jfreechart.zip && \
-     wget -cO rxtx.zip "http://rxtx.qbang.org/pub/rxtx/rxtx-2.1-7-bins-r2.zip" && \
-     unzip rxtx.zip && \
-     rm -f rxtx.zip && \
+     wget -cO ivy.zip https://dlcdn.apache.org//ant/ivy/2.5.1/apache-ivy-2.5.1-bin-with-deps.zip &&\
+     unzip ivy.zip && \
+     rm -f ivy.zip && \
      apk del wget
 
 WORKDIR /usr/src/app
 COPY . .
 
-RUN ant compile
-RUN ant jar
+RUN ant compile -lib ../apache-ivy-2.5.1/ivy-2.5.1.jar
+RUN ant jar -lib ../apache-ivy-2.5.1/ivy-2.5.1.jar
+
+
+WORKDIR /usr/src/app/
+RUN ls /usr/src/app/dist/
+CMD java -jar /usr/src/app/dist/ModbusPal.jar -loadFile=/usr/src/app/TestFacility.xmpp -hide
```

Comment:
Remove these two lines - the `ls` is just testing and the command will be defined by whoever extends the repository